inference-engine/src/mkldnn_plugin/nodes/mkldnn_region_yolo_node.h

```diff
@@ -49,6 +49,10 @@ class MKLDNNRegionYoloNode : public MKLDNNNode {
 
     static bool isSupportedOperation(const std::shared_ptr<const ngraph::Node>& op, std::string& errorMessage) noexcept;
 
+protected:
+    void prepareParams() override;
```

Comment:
You should just override `needPrepareParams` to return false. Because you don't need prepare params for each new shapes. So you may not override `prepareParams()`. `needPrepareParams` is enough

Moreover you should update last input dims in the end of `createPrimitive` if shapes are defined. [Example](https://github.com/v-Golubev/openvino/blob/1e19e7f74f6d39fc00c472d2ce3ddca791cecc17/inference-engine/src/mkldnn_plugin/nodes/mkldnn_cum_sum_node.cpp#L291)

Reply:
done, thanks